src/main/resources/config/liquibase/changelog/20230525144705_added_entity_FdaDrug.xml

```diff
@@ -17,8 +17,14 @@
             <column name="application_number" type="varchar(255)">
                 <constraints nullable="false" unique="true" uniqueConstraintName="ux_fda_drug__application_number" />
             </column>
+            <column name="sponsor_name" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+            <column name="overall_marketing_status" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
             <column name="drug_id" type="bigint">
-                <constraints nullable="true" unique="true" uniqueConstraintName="ux_fda_drug__drug_id" />
```

Comment:
It was originally one to one, however I eventually realized a generic drug can have multiple FDA approval because multiple pharma can manufacture it. 
For instance Imatinib has 12 manufactures based on open fda https://api.fda.gov/drug/drugsfda.json?search=openfda.generic_name:%22Imatinib%22&limit=20